docs/resources/aws_vpn_connections.md

```diff
@@ -0,0 +1,67 @@
+---
+title: About the aws_vpn_connections Resource
+platform: aws
+---
+
+# aws\_vpn\_connections
+
+Use the `aws_vpn_connections` InSpec audit resource to test properties of all AWS VPN Connections.
+
+## Syntax
+
+Ensure that one or more VPN Connections exist.
+
+    describe aws_vpn_connections do
+      it { should exist }
+    end
+
+## Parameters
+
+This resource does not expect any parameters.
+
+See also the [AWS documentation on VPN Connections](https://docs.aws.amazon.com/vpc/latest/userguide/vpn-connections.html).
+
+## Properties
+
+|Property           | Description|
+| ---               | --- |
+|vpn\_connection\_id    | This property provides all the IDs of the VPN Connections |
+|vpn\_gateway\_id       | This property provides all the IDs of the virtual private gateways associated with the VPN Connections |
+|tunnel\_options        | This property provides the tunnel options of the VPN Connections. |
+|state                  | This property provides the current state of the VPN Connections |
+
+## Examples
+
+##### Ensure that VPN connections are available.
+    describe aws_vpn_connections do
+      its('state') { should eq 'available' }
+    end
+
+##### Check tags    
+    describe aws_vpn_connections do
+      its('tags') { should include(:Name => 'vpn-connection-example-123')}
```

Comment:
This should be updated as `its('tags') { should include('Name' => 'vpn-connection-example-123')}` since the tag keys are in String format.

Apart from that, using plural resources is not the most convenient way of testing if a resource has a certain tag defined. If there is any other tags defined along with the queried one, this test will fail. It compares the given tag with the entire set of tags for each resource. 

Updating example as:
```
describe aws_vpn_connections.where { tags["Name"] == "vpn-connection-example-123" } do
  it { should exist }
  its('count') { should be 3 }
end
```
might demonstrate the usage of plural resources better.